### DISCUSSION TITLE
Module federation nuxt 3

### DISCUSSION THREAD

**Discussion by PouryaNoufallah96** [nuxt/nuxt#15899] [Questions]
hi. can someone show me a real working sample of  module federation in nuxt 3 for micro frontend development?
every sample ive seen had issues

**Comment by Baroshem** [comment]
Hey, 

Do you mean something like this? 

https://dev.to/jacobandrewsky/using-modules-and-pinia-to-structure-nuxt-3-app-5963

**Comment by PouryaNoufallah96** [comment]
hey,

no i mean module federation plugin for micro frontend . like remote and host separate applications.

i mean like this https://github.com/hlassiege/nuxt-vite-federation-plugin
but it does not work

**Reply by Baroshem** [reply]
I see. I dont have any resources then to help you with that :(

Maybe someone else will be able to share some light on this topic

**Comment by marcel0ll** [comment]
Does Nuxt 3 officially support module federation either with Webpack or Vite?

**Reply by PouryaNoufallah96** [reply]
i dont think so. there's a package for it I think => @originjs/vite-plugin-federation

**Reply by marcel0ll** [reply]
That is for vue3 + vite, but not specifically for Nuxt 3 as far as I understood.

**Reply by PouryaNoufallah96** [reply]
yea but we can use it. i reproduce it here https://github.com/PouryaNoufallah96/nuxt3-microfe
it has some errors but I think it can be fixed
